librz/asm/arch/8051/8051_ass.c

```diff
@@ -1,3 +1,4 @@
+// SPDX-FileCopyrightText: 2009-2019 hmht <55194419+hmht@users.noreply.github.com>
```

Comment:
Maybe remove all `users.noreply.github.com>` email occurencies? They don't make sense in copyright

Reply:
agreed